docs/conflux-portal/API_Reference/Signing_Data.md

```diff
@@ -1,15 +1,15 @@
 ---
 id: signing_data_with_portal
-title: 使用Conflux Portal浏览器插件钱包对数据签名
+title: 使用 Portal 浏览器插件钱包对数据签名
```

Comment:
suggestion:  'Portal' can be replaced with 'ConfluxPortal', thanks for your effort.

Reply:
ok